<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWindow</class>
 <widget class="QWidget" name="CameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
border: 2px solid line black;
	background-color: rgb(148, 134, 115);
	font: 14pt &quot;Sans Serif&quot;;
}
QLineEdit
{
border: 2px solid line black;
	
	background-color: rgb(227, 227, 227);
}

QLabel
{
border: 0px solid line black;
	
	background-color: rgb(148, 134, 115);

}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Картинки/Камера.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelFrameRateROI">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
border: 2px solid line black;
	
	background-color: rgb(211, 211, 158);
}</string>
     </property>
     <property name="text">
      <string> Строб  -  100 мс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
border: 2px solid line black;
	
	background-color: rgb(255, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>Камера</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelFrameRate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777213</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
border: 2px solid line black;
	
	background-color: rgb(211, 211, 158);
}</string>
     </property>
     <property name="text">
      <string>Кадры  - 100  мс</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkWorkBlock">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox
{
border: 2px solid line black;
	
	
	background-color: rgb(208, 0, 0);
}

QCheckBox::checked
{
border: 2px solid line black;
	
	background-color: rgb(85, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>Работа</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
